backend/wmg/pipeline/summary_cubes/calculate_markers.py

```diff
@@ -69,14 +110,12 @@ def wrapped(*args, **kwargs):
 # are not typically hashable.
 @_make_hashable
 @lru_cache(maxsize=None)
-def _query_tiledb_context_memoized(
-    context_filters: dict,
-    group_by_dims,
-    keep_dataset_ids,
+def _query_tiledb_context(
```

Comment:
no need for wrapper around memoization since this module is simplified to expect only cell type, tissue, and organism